src/sentry/scim/endpoints/members.py

```diff
@@ -92,24 +91,7 @@ def _should_delete_member(self, operation):
         parameters=[GLOBAL_PARAMS.ORG_SLUG, SCIM_PARAMS.MEMBER_ID],
         request=None,
         responses={
-            200: inline_serializer(
-                "SCIMMember",
-                fields={
-                    "schemas": serializers.CharField(),
-                    "id": serializers.CharField(),
-                    "userName": serializers.CharField(),
-                    "emails": inline_serializer(
-                        "SCIMMemberEmails",
-                        fields={
-                            "primary": serializers.BooleanField(),
-                            "value": serializers.CharField(),
-                            "type": serializers.CharField(),
-                        },
-                        many=True,
-                    ),
-                },
-                many=True,
-            ),
+            200: OrganizationMemberSCIMSerializer,
             401: RESPONSE_UNAUTHORIZED,
             403: RESPONSE_FORBIDDEN,
             404: RESPONSE_NOTFOUND,
```

Comment:
Do we need to declare this for all endpoints? Shouldn't this be standardized?

Reply:
Right now yes, since it requires a bit of customization with the framework: https://github.com/tfranzel/drf-spectacular/issues/334

It's simple and I can do it in a follow up PR. TODO added for now.

Reply:
I attempted to implement a solution for this quickly and ran into some issues with the current way we're using the library -- for now, we'll have to declare the standard responses if we want them to be known to the OpenAPI schema, but something that can be optimized with our use case as we go on.

If people forget to add them, it isn't the end of the world. It is also hard to know if for example, an API can 404 or not, and ideally we only add in error codes that are possible for an endpoint.